content/guide/dsl/supported-networks.mdx

```diff
@@ -106,6 +106,15 @@ description: Networks and protocols supported by the RSS3 Network DSL.
 			6
 		</td>
 	</tr>
+
```

Comment:
should not be added manually. will updated through the script: https://github.com/RSS3-Network/Docs/blob/main/scripts/generate-dsl-info.mjs

so just need to update on vsl networkParams contract.